Gems/Atom/Tools/MaterialCanvas/Code/Source/Window/MaterialCanvasMainWindow.cpp

```diff
@@ -173,6 +173,11 @@ namespace MaterialCanvas
                   "require clearing the cache to regenerate shaders for the new RHI.\n\nThe settings files containing the overrides will be "
                   "placed in the user/Registry folder for the current project.",
                   false),
+              AtomToolsFramework::CreateSettingsPropertyValue(
+                  "/O3DE/Atom/MaterialCanvas/ForceDeleteGeneratedFiles",
```

Comment:
Nit: Feels like this key should be a constant declared somewhere so that it can get reused between here and MaterialGraphCompiler, instead of relying on them keeping their two strings in sync.